variables.tf

```
@@ -7,6 +7,36 @@ variable "additional_auditing_trail" {
   description = "CloudTrail configuration for additional auditing trail"
 }
 
+variable "aws_account_password_policy" {
+  type = object({
+    allow_users_to_change        = bool
+    max_age                      = number
+    minimum_length               = number
+    require_lowercase_characters = bool
+    require_numbers              = bool
+    require_symbols              = bool
+    require_uppercase_characters = bool
+    reuse_prevention_history     = number
+  })
+  default = {
+    allow_users_to_change        = true
+    max_age                      = 90
+    minimum_length               = 14
+    require_lowercase_characters = true
+    require_numbers              = true
+    require_symbols              = true
+    require_uppercase_characters = true
+    reuse_prevention_history     = 24
+  }
+  description = "AWS account password policy parameters for the audit, logging and master account"
+}
+
+variable "aws_create_account_password_policy" {
+  type        = bool
+  default     = true
+  description = "Set to false to prevent the AWS account policy from being created in the audit, logging and master account"
```

I think it reads better if we use, the positive, `set to true`:

```suggestion
  description = "Set to true to create the IAM password policy in the audit, logging and master accounts"
```

What do you think?

---

Agreed